### PR TITLE
Added text-utilities component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added text-utilities component

### CODE REVIEW

**Review by Tanay-J — APPROVED** [review]
Approving the changes with couple of suggestions:
1. Utility class naming can be more consistent like either using ```mx-1```, ```mx-2``` etc or ```mx-xs```, ```mx-s```, using both types of naming conventions can get confusing at times. 
2. Also, try to write similar utility classes grouped together one after another, like all classes related to horizontal margin together. This will improve readability and prevent code repetition.

File: html/docs.html [hunk 1]
```diff
@@ -484,7 +484,35 @@ <h2 class="medium-heading docs-heading" id="input">
...
+                  Text Utilities
+                </h2>
+                <p class="mx-2 fs-ml mt-3 lh-25p">
+                  Text Utilities component can be used for better typography
+                </p>
+                <h2 class="medium-heading docs-heading" id="input">
+                  Headings
+                </h2>
+                <div class="sample-container m-2">
+                  <figure>
```
Comment (GuptaEesh) [inline html/docs.html:498]: You can instead use section tag here.

**Review by GuptaEesh — COMMENTED** [review]
The comments from the previous reviewer and a suggestion that I gave can be taken into consideration. The rest looks good.

**Comment by nareshpoonia** [comment]
> Approving the changes with couple of suggestions:
> 
> 1. Utility class naming can be more consistent like either using `mx-1`, `mx-2` etc or `mx-xs`, `mx-s`, using both types of naming conventions can get confusing at times.
> 2. Also, try to write similar utility classes grouped together one after another, like all classes related to horizontal margin together. This will improve readability and prevent code repetition.

Thank you for your comment, two type of naming convention is used for covering a lot of margin values in the utilities class using either one of them was limiting the number of values i can cover, will try to figure out a solution of this.

File: html/docs.html (hunk 1 above)
Comment (nareshpoonia) [reply html/docs.html:498]: Changed div to section, change sequence of utility classes.

**Pull request merged**